### PR TITLE
Add RipMe to up-for-grabs

### PR DESCRIPTION
**Pull request by metaprime**
See https://github.com/4pr0n/ripme/issues/556

### CODE REVIEW

File: _data/projects/ripme.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- archiver
+- reddit
+- imgur
+- deviantart
+- twitter
+- instagram
+- tumblr
+upforgrabs:
+  name: up-for-grabs
+link: https://github.com/4pr0n/ripme/labels/up-for-grabs
```
Comment (shiftkey) [inline _data/projects/ripme.yml]: This needs to be indented

Comment (metaprime) [reply _data/projects/ripme.yml]: Sorry about that. Will fix.

Comment (metaprime) [reply _data/projects/ripme.yml]: Fixed in latest.

**Comment by shiftkey** [comment]
@metaprime thanks!

**Pull request merged**